CMSgov/design-system
# [WNMGDS-1592] component page support

## Summary

Added support in doc site page headers to have things like
* Badge for page status (mostly used for components in draft phase)
* "Jump to guidance" link
* A link to related uswds guidance
These things are all configured per content page in the frontmatter of the page

## How to test

`yarn start:gatsby`. Navigate to Component > Accordion to see result

<img width="1787" alt="Screen Shot 2022-05-16 at 10 31 28 AM" src="https://user-images.githubusercontent.com/33579665/168641224-18029727-3f66-4a2b-ba7b-44e53edc86a8.png">
